setup.py

```diff
@@ -5,7 +5,7 @@
 long_description = "\n\n".join([open("README.rst").read()])
 
 install_requires = [
-    "threedi-schema==0.214.*",
+    "threedi-schema==0.216.*,>=0.216.1",
```

Comment:
iirc we generally pin on the minor version because we use it for breaking changes, so future schema updates might break the current modelchecker version